### PR TITLE
attributized filter_users and filter_groups configuration options

### LINKED ISSUE #11: add attribute to add/change [nss] filter_users
The template for the ssd.conf file does not provide a way to add or chnage the filter_users list.  Would it be possible to add an attribute (or perhaps two, a base_users list and an extra_users list) so that list coudl be customized?

### PR DESCRIPTION
**Pull request by Poohblah**
fixes #11

### CODE REVIEW

File: templates/default/sssd.conf.erb [hunk 1]
```diff
@@ -4,7 +4,8 @@ services = nss, pam<% if node['sssd_ldap']['ldap_autofs'] %>, autofs<% end %><%
 domains = LDAP
 
 [nss]
-filter_users = root,named,avahi,haldaemon,dbus,radiusd,news,nscd
+filter_users = <%= node['sssd_ldap']['filter_users'].join(',') %>
+filter_groups = <%= node['sssd_ldap']['filter_groups'].join(',') %>
```
Comment (tas50) [inline templates/default/sssd.conf.erb]: How does this behave if the array is empty and we end up with 'filter_groups = '?

Comment (Poohblah) [inline templates/default/sssd.conf.erb]: SSSD ignores configuration options that have empty values.  In this case, the default value for `filter_users` is just `root` and the default value for `filter_groups` is empty.

Comment (Poohblah) [inline templates/default/sssd.conf.erb]: I take that back.  It seems to depend on the type of argument the configuration directive takes.  For instance, an empty value for a configuration option that expects a string resolves to an empty string while an empty value for a configuration option that expects a boolean appears to be ignored. The `sssd.conf(5)` man page unfortunately is not clear on this issue.

With regards to `filter_groups = ` in particular, this is equivalent to `filter_groups = ""`, in other words, do not filter any groups.

**Comment by tas50** [comment]
Works for me. Thanks for the info

**Pull request merged**